Makefile

```diff
@@ -58,12 +58,14 @@ test-compile-changed: os-autoinst/
 
 .PHONY: test-yaml-valid
 test-yaml-valid:
-	@# Get list of changed yaml files
-	$(eval YAMLS=$(shell sh -c "git --no-pager diff --diff-filter=dr --name-only master | grep '\(schedule\|test_data\)/.*\.y.\?ml$$'"))
-	if test -n "$(YAMLS)"; then \
-	  export PERL5LIB=${PERL5LIB_} ; tools/test_yaml_valid $(YAMLS);\
+	@# Get list of changed yaml files. We only want to lint changed files for
+	@# now because yamllint complains about a lot of existing files
```

Comment:
Can you re-phrase that? Should that be a TODO? Is this about limiting the files you are fixing now?

Reply:
Before only changed files were validated with `test_yaml_valid` and linted with yamllint.
Now we validate all files, but still we can't run yamllint on all files because there are too many errors to fix in this PR.
This should be done in a followup IMHO.
Any suggestion for rephrasing?

Reply:
Okay, I see the intention behind it... how about this:

```sh
@# Lint files that have been changed only
@# TODO: Update all existing files to conform to the schema
```

Reply:
If I use the word 'TODO' then @okurz will reject the PR ;-)

Also the message "Update all existing files to conform to the schema" is wrong. They conform to the schema but they don't conform to the yamllint rules.

I would like to repeat that it was like that before, and I didn't change that.

Reply:
Leave it as is. It just results in slight confusion.